pkg/cri/streaming/wsstream/conn.go

```diff
@@ -256,6 +364,21 @@ func (conn *Conn) handle(ws *websocket.Conn) {
 		if len(data) == 0 {
 			continue
 		}
+		if supportsStreamClose && data[0] == StreamClose {
```

Comment:
path seems to allow for the prior version without close.. protocol!  


ok good

Reply:
✅